### DISCUSSION TITLE
How-To delete user

### DISCUSSION THREAD

**Discussion by cupcakearmy** [dani-garcia/vaultwarden#1279] [Q&A] [answered]
Hey there, 

Quick question: How can I delete a user? i mean not from an org, but as a whole. That someone forgot the password and I need to delete that account xD

**Comment by cupcakearmy** [comment, marked as answer]
NVM There is an admin site https://github.com/dani-garcia/bitwarden_rs/wiki/Enabling-admin-page

**Reply by HanSolo72** [reply]
I have a user that can't be deleted. I made a screenshot with changed names: https://i.imgur.com/nzIIFlJ.png

The user "john smith" from this screenshot can't be deleted. I have to put in the e-mail address, but it's always wrong, wheter I put in only the e-mail, or the whole string including the name and the <> symbols.

As example for a normal user I show also Jane Doe. I think there's a error in the user database. How can I fix this error?

**Reply by BlackDex** [reply]
![image](https://github.com/dani-garcia/vaultwarden/assets/610450/aa656929-306b-45b2-a514-54a1e0afeba9)

I pasted the image just to make it easier to see.
For one, strange how that even gets into the database, that is something to look into.
And, there might be an issue with deleting a user via the admin panel with such names. I just encountered something similar my self yesterday, and have not yet looked into it.
